domain/src/main/java/com/example/domain/repositories/DeleteOldWeatherDataRepository.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.domain.repositories
+
+interface DeleteOldWeatherDataRepository {
```

Comment:
There are three repositories:
CurrentWeatherRepository
DailyWeatherRepository
DeleteOldWeatherDataRepository

All of them work with weather data.  Let's merge them into one WheatherRepository to simplify codebase. 
For one operation we should create separate UseCase, but not separate repository.

Reply:
Fixed